identity/backend/src/main/java/io/camunda/identity/usermanagement/service/UserService.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.identity.usermanagement.service;
+
+import io.camunda.identity.usermanagement.CamundaUser;
+import io.camunda.identity.usermanagement.CamundaUserWithPassword;
+import java.util.List;
+import org.springframework.dao.DuplicateKeyException;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserService {
+  private final CamundaUserDetailsManager userDetailsManager;
+
+  private final PasswordEncoder passwordEncoder;
+
+  public UserService(
+      final CamundaUserDetailsManager userDetailsManager, final PasswordEncoder passwordEncoder) {
+    this.userDetailsManager = userDetailsManager;
+    this.passwordEncoder = passwordEncoder;
+  }
+
+  public CamundaUser createUser(final CamundaUserWithPassword userWithCredential) {
+    try {
+      final UserDetails userDetails =
+          org.springframework.security.core.userdetails.User.withUsername(
+                  userWithCredential.user().username())
+              .password(userWithCredential.password())
+              .passwordEncoder(passwordEncoder::encode)
+              .disabled(!userWithCredential.user().enabled())
+              .roles("DEFAULT_USER")
+              .build();
+      userDetailsManager.createUser(userDetails);
+      return new CamundaUser(userDetails.getUsername(), userDetails.isEnabled());
```

Comment:
❓ NABD for now but here I think we'd want to return the object stored in the database so we also get the UUID for the user, we could use `fundUserByUsername` here for that or use a new SQL and add `returning ....` to the end of the statement to make it a single statement.